docs/collecting-data/collecting-from-own-applications/mobile-trackers/previous-versions/v3/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Mobile Trackers v3.0"
+title: "Mobile Trackers v3.x"
```

Comment:
I would move this file to `/previous-versions/mobile-trackers-v3-0/` for consistency with `/previous-versions/mobile-trackers-v2-0/`

Reply:
Done